app/javascript/geoblacklight/leaflet/controls/sleep.js

```diff
@@ -0,0 +1,78 @@
+import { DomUtil, DomEvent, Handler } from "leaflet";
+
+export default class Sleep extends Handler {
+  addHooks() {
+    const container = DomUtil.create("div", "wake-overlay w-100 h-100 position-absolute align-items-center justify-content-center", this._map._container);
+    const button = DomUtil.create("button", "btn btn-primary wake-button", container)
+
+    button.innerHTML = this._map.options.MESSAGE;
+    button.setAttribute("tabindex", "0");
+    button.style.zIndex = 2000;
+    container.style.zIndex = 1999;
+    const background = String(this._map.options.BACKGROUND);
+    container.style.background = background;
+    this.container = container;
+    this.sleepMap();
+    this.setListeners(this._map, container)
```

Comment:
in one case you pass `this._map` and in the other you don't, and keep calling it (lines 20-25). Maybe use the same way in both methods?

Reply:
Good catch. This was cruft from another way I was looking at doing it.